pkgs/by-name/th/themix-gui/package.nix

```diff
@@ -0,0 +1,80 @@
+{ lib
+, stdenv
+, sassc
+, gdk-pixbuf
+, glib
+, gobject-introspection
+, librsvg
+, gtk3
+, python3
+, fetchFromGitHub
+, wrapGAppsHook
+}:
+
+let
+  py = python3.withPackages (p: [
+    p.pygobject3
+  ]);
+in
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Reply:
Recursive attributes in `stdenv.mkDerivation` should be done with the `finalAttrs` pattern as described here https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes

Reply:
Yeah everyone has their own opinion on this one. It's less relevant here as it's only used for the fetcher and not in the actual build, i.e. fetcher has to be overridden regardless. Howevere, using `finalAttrs` introduces an eval time penalty that many argue is not worth it.

Reply:
I can convert it to a `let`-expr if that reads better.